### PR TITLE
:mortar_board: If Else Topic Compounding Expressions

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Show an example of an `if` with more complex compound expressions. 

### Testing
Yersh 

### Additional Notes
Next topic will cover `else`

### CODE REVIEW

**Comment by twentylemon** [comment]
Diff is a mess. I'm guessing you inadvertantly changed line endings. Check git settings?

**Comment by jameshughes89** [comment]
oh god, the merge conflicts.

File: site/topics/if-else/if-else.rst [hunk 1]
```diff
@@ -1,278 +1,345 @@
...
+
+    * Since the expression evaluated to ``False``, the indented code is skipped
+    * Return ``"Nope"``
+
+
+.. raw:: html
+
+	<iframe width="560" height="315" src="https://www.youtube.com/embed/8OAsHilB0jw" frameborder="0" allowfullscreen></iframe>
+   
+
```
[the remark this answers is not in the document]
Comment (jameshughes89) [reply site/topics/if-else/if-else.rst:171]: Compound Conditions

[the remark this answers is not in the document]
Comment (twentylemon) [reply site/topics/if-else/if-else.rst:171]: YOLO

**Review by twentylemon — APPROVED** [review]

**Pull request merged**